codesandbox/codesandbox-client
# feat(pro): New subscription page

### New pro page

#### How to test

You will need the following workspaces / accounts:

- Team workspace (as admin) **without** Pro subscription
- Team workspace (as admin) **with** Pro subscription
- Personal workspace **without** Pro subscription
- Personal workspace **with** Pro subscription
- Personal workspace with **legacy** Pro subscription

For each workspace, visit the `/pro` page. Make sure that the following works:
- [ ] Workspace picker works
- [ ] Shows right title
- [ ] Shows proper features in cards
- [ ] Checkout links work
- [ ] Manage workspace links work
- [ ] Contact us link works

### Notes

I haven't gotten to the legacy pro pages yet. I don't want to make this PR bigger than it is. We can update it in another PR, as there is a lot to clean up in there too.

### Screenshots

No subscription
<img width="1624" alt="Screenshot 2022-11-29 at 10 19 03" src="https://user-images.githubusercontent.com/7533849/204489247-5caa6558-62ad-480f-97ac-9a1f169db9d7.png">

Pro subscription
<img width="1624" alt="Screenshot 2022-11-29 at 10 20 34" src="https://user-images.githubusercontent.com/7533849/204489520-156fa31a-75e5-4eb4-9682-4ff1f2d0a3e2.png">

Workspace picker
<img width="1624" alt="Screenshot 2022-11-29 at 10 22 00" src="https://user-images.githubusercontent.com/7533849/204489898-cf4127b8-a7bb-4de1-b1f8-2bb6adce0de1.png">

Personal no subscription
<img width="1624" alt="Screenshot 2022-11-29 at 10 53 26" src="https://user-images.githubusercontent.com/7533849/204497202-1ad0754d-30bb-4314-b5fa-8bbac009bf50.png">

Personal pro
<img width="1624" alt="Screenshot 2022-11-29 at 10 54 46" src="https://user-images.githubusercontent.com/7533849/204497381-f7e19cdc-cfc7-4f24-b680-6f02c116a475.png">

Legacy pro
<img width="1624" alt="Screenshot 2022-11-29 at 10 24 20" src="https://user-images.githubusercontent.com/7533849/204490351-29bfe373-d662-4c5d-bf88-a120bedf22df.png">
